client/components/tinymce/plugins/insert-menu/plugin.jsx

```diff
@@ -26,11 +27,48 @@ const initialize = editor => {
 		cmd: menuItems[ 0 ].cmd,
 		menu: menuItems.map( ( { name } ) => editor.menuItems[ name ] ),
 		onPostRender() {
-			ReactDOM.render(
-				<Gridicon icon="add-outline" />,
-				this.$el[ 0 ].children[ 0 ]
-			);
-		}
+			ReactDOM.render( <Gridicon icon="add-outline" />, this.$el[ 0 ].children[ 0 ] );
+
+			// Listen to `mouseenter` events on the (+) part of the Inserter menu to show
+			// the "Insert content" tooltip.
+			const insertContentElm = this.$el[ 0 ].children[ 0 ];
+
+			insertContentElm.addEventListener( 'mouseenter', () => {
+				// We need to select the tooltip during the `mouseenter` event and not outside.
+				// Otherwise, Tinymce renders an empty tooltip somewhere in the editor.
+				const btnTooltip = this.tooltip();
+
+				btnTooltip.text( i18n.translate( 'Insert content' ) );
+
+				btnTooltip.moveBy( -10, 0 );
```

Comment:
These -10 and 24 constants seem to be fine-tuned to your OS, screen and fonts. On my screen, the tooltips are slightly off-center.

Have you considered using `btnTooltip.moveRel( insertContentElm, 'bc-tc' )` to align the tooltip? It seems to work better than `moveTo` for me. And you could remove the `mouseleave` listeners completely.

Even `moveRel` requires some fine tuning to make it perfect (vertical alignment is one pixel off, it should respect the `bc-tr` and `bc-tl` options), but you must watch very closely to spot these glitches.

I'm going to approve this now so that you can merge -- it's up to you whether you want to use `moveRel` now, later, or never.